client/app/scripts/superdesk/filters.js

```diff
@@ -221,5 +221,12 @@
                 var conditionValue = labels.length > 0 ? labels.join(', ') : filterCondition.value;
                 return '(' + filterCondition.field + ' ' + filterCondition.operator + ' ' + conditionValue + ')';
             };
+        })
+        .filter('formatString', function() {
```

Comment:
can we think about a better name? it doesn't do much formatting.
it's more like making it human readable.. but `h` might be too short :smile: